tests/raw_to_standard/planned_events_test.py

```diff
@@ -215,3 +207,68 @@ def test_get_lanes_list_2():
         lane_closures_hex, num_lanes, closedLaneTypes))
     assert planned_events.get_lanes_list(
         lane_closures_hex, num_lanes, closedLaneTypes) == expected
+
+
+def test_is_incident_true_true():
+    msg = {
+        "properties": {
+            "id": "OpenTMS-Incident2028603626",
+            "type": "Emergency Roadwork"
+        }
+    }
+    # is_incident, is_wz
+    expected = (True, True)
+    actual = planned_events.is_incident(msg)
+    assert actual == expected
+
+
+def test_is_incident_true_false():
+    msg = {
+        "properties": {
+            "id": "OpenTMS-Incident2028603626",
+            "type": "Chain Law Code 18"
+        }
+    }
+    expected = (True, False)
+    actual = planned_events.is_incident(msg)
+    assert actual == expected
+
+
+def test_is_incident_false_false():
```

Comment:
Do we need a "_false_true" test in here to shore up?

Reply:
no, if it is a planned event, then the incident work zone type check doesn't make any sense anymore. All of the types for planned events are handled by the EVENT_TYPE_MAPPING dict